package.json

```diff
@@ -35,15 +35,15 @@
     "eslint-plugin-standard": "^4.0.0",
     "husky": "^1.3.1",
     "lint-staged": "^8.1.0",
-    "parcel-bundler": "^1.11.0",
+    "parcel-bundler": "^1.12.3",
     "posthtml-include": "^1.2.0",
     "prettier": "^1.15.3",
     "pretty-quick": "^1.8.0",
     "rimraf": "^2.6.3",
     "sass": "^1.15.3"
   },
   "dependencies": {
-    "jquery": "^3.3.1",
+    "jquery": "^3.4.1",
```

Comment:
Why you update dependencies in this commit?

Reply:
i think it can be accident. I resolved and commited this change.

Reply:
avoid this in the future